pkg/machines/libvirt.es6

```diff
@@ -170,29 +175,47 @@ LIBVIRT_PROVIDER = {
         };
     },
 
+    INIT_DATA_RETRIEVAL () {
+        logDebug(`${this.name}.INIT_DATA_RETRIEVAL():`);
+        return dispatch => {
+            dispatch(getOsInfoList());
+            return cockpit.script(getLibvirtServiceNameScript, null, { err: "message", environ: ['LC_ALL=en_US.UTF-8'] })
```

Comment:
As this script is now an one-liner, and you already do some regexp post-processing below, you could just simply call `cockpit.spawn()` here on `systemctl --no-legend list-unit-files libvirtd.service libvirt-bin.service` and ditch the script. This makes the code easier to read as everything that happens here is at the same place.

Reply:
I think we still need the script.